installers/npm/binary.js

```diff
@@ -212,13 +212,19 @@ class Binary {
   }
 }
 
-const getBinary = () => {
+const getBinary = (overrideInstallDirectory = "") => {
   const platform = getPlatform();
   const download_host = process.env.npm_config_apollo_rover_download_host || process.env.APOLLO_ROVER_DOWNLOAD_HOST || 'https://rover.apollo.dev'
   // the url for this binary is constructed from values in `package.json`
   // https://rover.apollo.dev/tar/rover/x86_64-unknown-linux-gnu/v0.4.8
   const url = `${download_host}/tar/${name}/${platform.RUST_TARGET}/v${version}`;
-  let binary = new Binary(platform.BINARY_NAME, url);
+  const { dirname } = require('path');
+  const appDir = dirname(require.main.filename);
+  let installDirectory = join(appDir, "binary");
+  if (overrideInstallDirectory) {
```

Comment:
nit: this works, but because of my js-induced type paranoia, this would feel marginally better as `overrideInstallDirectory != null && overrideInstallDirectory !== ""` in conjunction with the above comment